### PR TITLE
Shipyard release for 0.8.0-pre0

### PR DESCRIPTION
**Pull request by mkolesnik**
Testing the semi-auto release process, starting with Shipyard

Signed-off-by: Mike Kolesnik <mkolesni@redhat.com>

### CODE REVIEW

File: releases/v0.8.0-pre0.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
```
Comment (mangelajo) [inline releases/v0.8.0-pre0.yaml:1]: may be we should call this v0.8.0-shipyard-pre0.yaml ? 

So we can then keep the other items in the repo?

Comment (mangelajo) [reply releases/v0.8.0-pre0.yaml:1]: In a way that we would keep:

- v0.8.0-pre0-shipyard
- v0.8.0-pre0-admiral
- v0.8.0-pre0-submariner-lighthouse
- v0.8.0-pre0-release

?

Comment (mkolesnik) [reply releases/v0.8.0-pre0.yaml:1]: The idea is to continuously update the same file until its all released.

Comment (mkolesnik) [reply releases/v0.8.0-pre0.yaml:1]: I don't see a point in keeping separate files, its just different stages of the same release process. Once you've finished the previous stage you continue to the next one (and don't update the hash of the already released parts)

Comment (mangelajo) [reply releases/v0.8.0-pre0.yaml:1]: Yeah, but for documentation purposes, I think it's helpful to keep, at least for now, so the process is fully traceable. We could do that via git logs too.

What you say also makes sense I guess :)

File: releases/v0.8.0-pre0.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+version: v0.8.0-pre0
+name: 0.8.0-pre0
+pre-release: true
+# Contents of release notes should be markdown formatted as that's what
+# GitHub uses
+release-notes: |
+  Here be dragons
+status: shipyard
+components:
+  shipyard: 7e63ce8
```
Comment (mangelajo) [inline releases/v0.8.0-pre0.yaml:11]:
```suggestion
  shipyard: 5e8ab3e
```

? :)

**Comment by mangelajo** [comment]
:man_dancing:

File: releases/v0.8.0-pre0.yaml (hunk 2 above)
Comment (mkolesnik) [reply releases/v0.8.0-pre0.yaml]: Done

**Review by mangelajo — APPROVED** [review]

**Review by sridhargaddam — APPROVED** [review]
Thanks @mkolesnik

**Pull request merged**